src/test/ui/lint/unused/unused-async.rs

```diff
@@ -30,10 +30,10 @@ impl Wowee {
 async fn test() {
     foo(); //~ ERROR unused return value of `foo` that must be used
     //~^ ERROR unused implementer of `Future` that must be used
-    foo().await; //~ ERROR unused awaited future returned by `foo` that must be used
+    foo().await; //~ ERROR unused output of future `foo` that must be used
```

Comment:
Mostly a nit, but I think a separating word between "future" and "`foo`" would be slightly more clear. My comment before used "from" but the original "returned by" also works.

Reply:
Changed to "unused output of future returned by `foo`"